README.md

```diff
@@ -61,12 +61,22 @@ before running this command.)
 
 At the moment, there is no automated nightly build process, but we occasionally
 build nightlies based on PyTorch nightlies by hand following the instructions in
-[packaging](packaging).  To install the latest nightly, run:
+[packaging](packaging).  To install the latest nightly via pip, run:
 
 ```
+pip install numpy
+pip install future # only on python 2.7
+pip install six    # only on python 3.5
```

Comment:
I'm surprised we have to add the dependencies explicitly :(

Reply:
not sure why pip didn't do that, my tests showed I have to install them manually.